snap/hooks/configure

```diff
@@ -385,6 +385,13 @@ then
   need_api_restart=true
 fi
 
+# Disable the watchlist feature until k8s-dqlite can handle it
+if ! grep -e "feature-gates=WatchList=false" ${SNAP_DATA}/args/kube-apiserver
+then
+  echo '--feature-gates=WatchList=false' >> ${SNAP_DATA}/args/kube-apiserver
```

Comment:
What if the user already has `--feature-gates` set? We would be overwriting his options.